src/core/maps/area_occupancy_estimator.h

```diff
@@ -6,32 +6,44 @@
 #include <type_traits>
 
 #include "cell_occupancy_estimator.h"
-
+/*!
+ * \brief Derived class from CellOccupancyEstimator to calculate the value of occupancy of one cell
+ *
+ * This class appears to make a rule to calculate the value of cell occupancy based on laser way through the cell.
```

Comment:
"<...> appears to make <...>" -- Don't you know exactly what is going on?

Consider a shorter version of description: "The estimation is based on a ration of cell's chunks that are produced by cutting the cell with a laser beam"